auth-server/src/main/java/com/antonio/authserver/dto/ParentRealmDto.java

```diff
@@ -0,0 +1,36 @@
+package com.antonio.authserver.dto;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+
+@Getter
+@Setter
+public class ParentRealmDto {
```

Comment:
nu aveti deja un dto pentru asta?

Reply:
@radasilviu lam redenumit in RealmRelationsDto ca sa fie mai sugestiv

Reply:
@radasilviu realmdto avea numa 1 way relation de la children, el nu field cu ei.